tools/lib/provision.py

```diff
@@ -186,7 +186,7 @@
                 "postgresql-server-dev-{0}",
             ]
         ]
-    elif codename == "buster":
+    elif codename == "10":
```

Comment:
Similarly, `elif vendor == "debian" and version == "10"`.

Reply:
6th commit.